pilot/cmd/pilot-discovery/main.go

```diff
@@ -156,6 +156,10 @@ func init() {
 	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.DiscoveryOptions.EnableCaching, "discoveryCache", true,
 		"Enable caching discovery service responses")
 
+	// SDS options
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.SdsArgs.SdsK8sTokenPath, "sdsK8sTokenPath", "",
```

Comment:
I'm not sure global settings is the way to go - and flags are the worse kind. 

At least mesh config can be reloaded without restarting pilot. 

Does it make sense to have a single sdsK8sTokenPath across all clusters ? Is there any value in making it customizable in the first place ?

Reply:
The value in making it customizable is: to run SDS tests and user guide, we can place an example SDS token to a customized path. Do you have suggestions on how to pass sdsK8sTokenPath?

Reply:
The sdsK8sTokenPath configuration does not need to be reloaded: it will be used in SDS tests and user guide.